var/spack/repos/builtin/packages/autoconf-archive/package.py

```diff
@@ -13,3 +13,11 @@ class AutoconfArchive(AutotoolsPackage, GNUMirrorPackage):
 
     version('2022.02.11', sha256='78a61b611e2eeb55a89e0398e0ce387bcaf57fe2dd53c6fe427130f777ad1e8c')
     version('2019.01.06', sha256='17195c833098da79de5778ee90948f4c5d90ed1a0cf8391b4ab348e2ec511e3f')
+
+    # https://github.com/autoconf-archive/autoconf-archive/pull/251
+    patch('2022.02.11-ax_cc_maxopt-nvhpc.patch', when='@2022.02.11')
```

Comment:
```suggestion
    patch('2022.02.11-ax_cc_maxopt-nvhpc.patch', when='@2022.02.11 %nvhpc')
```

Reply:
Does it matter if it is applied when it is not nvhpc? This is more a general bug fix than anything.

Reply:
That patch is from another pr that should be rebased away now. The bug is a missing `;;` in the bash `case` condition for `nvhpc` which in turns causes syntax errors for compilers unrelated to nvhpc, thus it needs to always be applied.

Reply:
Regardless though, it's rebased out of this pr now.